soh/soh/Enhancements/game-interactor/GameInteractor.cpp

```diff
@@ -45,6 +45,11 @@ bool GameInteractor::IsSaveLoaded() {
     return (gPlayState == NULL || player == NULL || gSaveContext.fileNum < 0 || gSaveContext.fileNum > 2) ? false : true;
 }
 
+extern "C" bool IsSaveLoadedWrapper() {
+    GameInteractor* gameInteractor = GameInteractor::Instance;
+    return gameInteractor->IsSaveLoaded();
+}
```

Comment:
i think following the naming patterns used for other extern c GI things might make sense here

https://github.com/HarbourMasters/Shipwright/blob/8a14fea94c07332bcf61a135076d930073e1d2da/soh/soh/Enhancements/game-interactor/GameInteractor_Hooks.h#L3-L8

https://github.com/HarbourMasters/Shipwright/blob/8a14fea94c07332bcf61a135076d930073e1d2da/soh/soh/Enhancements/game-interactor/GameInteractor_Hooks.cpp#L9-L11

so something like

```suggestion
extern "C" bool GameInteractor_IsSaveLoaded() {
    return GameInteractor::Instance->IsSaveLoaded();
}
```